google/cloud/storage/benchmarks/throughput_options.cc

```diff
@@ -0,0 +1,240 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "google/cloud/storage/benchmarks/throughput_options.h"
+#include "absl/strings/str_split.h"
+
+namespace google {
+namespace cloud {
+namespace storage_benchmarks {
+
+google::cloud::StatusOr<ThroughputOptions> ParseThroughputOptions(
+    std::vector<std::string> const& argv, std::string const& description) {
+  ThroughputOptions options;
+  bool wants_help = false;
+  bool wants_description = false;
+
+  auto parse_checksums = [](std::string const& val) -> std::vector<bool> {
+    if (val == "enabled") {
+      return {true};
+    }
+    if (val == "disabled") {
+      return {false};
+    }
+    if (val == "random") {
+      return {false, true};
+    }
+    return {};
+  };
+
+  std::vector<OptionDescriptor> desc{
```

Comment:
(no change necessary... just an FYI) Now that we have a dep on Abseil, we may want to consider using Abseil's flag parsing library, otherwise, we end up re-inventing a lot of standard flag parsing stuff. I know we already have a number of places where we parse flags ourselves, so I see no problem with adding more of it here in this PR (again, no change necessary). But going forward, we may want to consider Abseil flags.

https://abseil.io/docs/cpp/guides/flags